x-pack/solutions/observability/plugins/slo/kibana.jsonc

```diff
@@ -41,16 +36,16 @@
       "unifiedSearch",
       "usageCollection",
       "fieldsMetadata",
-      "apmSourcesAccess"
+      "apmSourcesAccess",
+      "spaces"
```

Comment:
This assumes that `spaces` will never be disabled (or it won't be disabled while SLO is enabled). Otherwise, Kibana will refuse to start.

Q: Is this a safe assumption?

cc @elastic/kibana-security

Reply:
to be on the safe side, I reverted the change and added a fallback when it is not defined